lib/src/block_syntaxes/fenced_code_block_syntax.dart

```diff
@@ -19,77 +18,104 @@ class FencedCodeBlockSyntax extends BlockSyntax {
   const FencedCodeBlockSyntax();
 
   @override
-  bool canParse(BlockParser parser) {
```

Comment:
Can you say what effect this has, removing `canParse`? Does it mean `canParse` will always return true?

Reply:
So the default `canParse` of  `BlockSyntax` applies:
```dart
bool canParse(BlockParser parser) {
  return pattern.hasMatch(parser.current);
}
```